client/src/styles/global.css

```diff
@@ -2,10 +2,12 @@
 /* SEE: ./README.md */
 
 /* SETTINGS */
-@import url('@tacc/core-styles/src/lib/_imports/settings/border.css');
```

Comment:
Some files, just loaded from `dist`[^1] instead of `src`.

[^1]: Yes, I store `/dist` in the repo. I stop that in [Core-Styles v3](https://github.com/orgs/TACC/projects/10). And NPM package will have `/dist` (and only `/dist`).